### PR TITLE
Adds a new grove planet away site, with some ghostroles

### PR DESCRIPTION
**Pull request by 4000daniel1**
A forested badlands planet which up to three people (ghostroles) crash landed on with escape pods. Since then they've been surviving and made basic shelter while waiting for rescue. Some distance away from the shelter, a spooky vannatusk (also a ghostrole) lives in a cave and may or may not decide to try to eat the aforementioned people.  Or become best friends with them. Or just observe them from a distance. I really don't know, and I think the players figuring that out might end up being the fun part.

Additionally, this PR ended up touching a bit more code than I expected it to. These include:

1. Made a new mob for the vannatusk ghostrole which is identical to the normal vannatusk except it DOESN'T automatically shoot bones at everything, has a bit more health, and can see in the dark.
2. Changed the greatworm mob to not spawn a space tile upon dying. Mostly just cause it made them impossible to feature in this away site, but also cause it didn't make that much sense in the first place.
3. Made diyaabs butcherable, for a small amount of meat.

While these are all basic things, I have ZERO idea what I'm doing and fully expect there to be typos and other not ideal stuff.

### CODE REVIEW

**Comment by 4000daniel1** [comment]
!review

File: code/modules/mob/living/simple_animal/hostile/vannatusk_ghostrole.dm [hunk 1]
```diff
@@ -0,0 +1,76 @@
+/mob/living/simple_animal/hostile/vannatusk_ghostrole
```
Comment (SleepyGemmy) [inline code/modules/mob/living/simple_animal/hostile/vannatusk_ghostrole.dm:1]: make this a subtype to cut down on copypaste.

Comment (4000daniel1) [reply code/modules/mob/living/simple_animal/hostile/vannatusk_ghostrole.dm:1]: Is there any easy way to do this while also having the subtype ignore the code that makes them shoot spikes? I tried using istype to make it possible but I couldn't get it to work properly.

Comment (SleepyGemmy) [reply code/modules/mob/living/simple_animal/hostile/vannatusk_ghostrole.dm:1]: override the procs.

File: code/modules/mob/living/simple_animal/hostile/vannatusk.dm [hunk 2]
```diff
@@ -45,6 +45,53 @@
 
 	psi_pingable = FALSE
 
+/mob/living/simple_animal/hostile/vannatusk/vannatusk_ghostrole
```
Comment (SleepyGemmy) [inline code/modules/mob/living/simple_animal/hostile/vannatusk.dm:48]:
```suggestion
/mob/living/simple_animal/hostile/vannatusk/ghostrole
```
dont forget other places that use the path.

**Comment by 4000daniel1** [comment]
OH BOY I GOOFED

**Comment by 4000daniel1** [comment]
I'll just make a new PR later I honestly don't know how I managed this.

**Pull request closed without merging**